serve/graph/schema.graphql

```diff
@@ -0,0 +1,194 @@
+"""
+Represents a blockchain block with various attributes detailing its creation and content.
+"""
+type Block {
```

Comment:
Curious to understand if you think it makes sense to reorganise this so that the transactions are contained within the block, (ie. Block contains `transactions: [Transaction]`), "unifying" the filter API. No strong feelings either way (I can see both ways having advantages/drawbacks), just curious to know if you thought of it and if so what you think of it.

Reply:
Blocks and transactions are queried using distinct gRPC endpoints when retrieved from the node. You have some information in the obtained Block structure, but it is not parsed, only represented as bytes.

I believe if the reader does not know the correct serialization format of these bytes, it would be safer to keep them separate until the API endpoint is improved.